soccer/writers.py

```diff
@@ -166,11 +167,12 @@ class Csv(BaseWriter):
     def generate_output(self, result):
         if not self.output_filename:
             for row in result:
-                click.echo(','.join(str(item) for item in row))
+                click.echo(u','.join(unicode(item) for item in row))
         else:
             with open(self.output_filename, 'w') as csv_file:
                 writer = csv.writer(csv_file)
                 for row in result:
+                    row = [unicode(s).encode('utf-8') for s in row]
```

Comment:
same here...why do u need to encode to 'utf-8' when u are already converting to unicode? I do not know much about the difference btw the two so curious :)